cmd/notary/tuf_test.go

```diff
@@ -33,7 +44,21 @@ func TestTokenAuth200Status(t *testing.T) {
 	s := httptest.NewServer(http.HandlerFunc(NotAuthorizedTestHandler))
 	defer s.Close()
 
-	auth, err := tokenAuth(s.URL, baseTransport, gun, readOnly)
+	auth, err := tokenAuth(s.URL, baseTransport, gun, readOnly, false)
+	require.NoError(t, err)
+	require.NotNil(t, auth)
+}
+
+func TestAdminTokenAuth200Status(t *testing.T) {
```

Comment:
Yeah, definitely - I was wondering how we could better test for this, and @endophage might have more context around the token auth code and tests in distribution.  I think a tracking issue would be good 👍